assets/dev/js/editor/utils/presets-factory.js

```diff
@@ -47,6 +47,11 @@ presetsFactory = {
 		return this.getPresets( parsedStructure.columnsCount, parsedStructure.presetIndex );
 	},
 
+	// Grid preset looks like 1-2 ( 1 rows, 2 columns )
+	getParsedGridStructure( selectedStructure ) {
+		return selectedStructure.split( '-' );
+	},
```

Comment:
@ArmanPurtoian 

Please add this line - same as the other function in the beginning of your function:
`selectedStructure += ''; // Make sure this is a string`
What do you think?

Then can you also rather return an object? Because to use `rowsAndCols[ 1 ]` below is a bit confusing. `parsedStructure.rows` and `parsedStructure.columns` would be better?